ext/crates/fp/src/limb.rs

```diff
@@ -97,7 +261,7 @@ pub(crate) fn reduce<const P: u32>(limb: Limb) -> Limb {
 
 /// Check whether or not a limb is reduced, i.e. whether every entry is a value in the range `0..P`.
 /// This is currently **not** faster than calling [`reduce`] directly.
-pub(crate) fn is_reduced<const P: u32>(limb: Limb) -> bool {
+pub(crate) fn _is_reduced<const P: u32>(limb: Limb) -> bool {
```

Comment:
What is the leading underscore for?

Reply:
It's only used in `_truncate`, which is itself never used. Without the underscore clippy complains about dead code. I believe it would be used in `BaseVectorMut::add_truncate`, but that one is unimplemented right now. I should probably fix that if we want to use `algebra::PolynomialAlgebra::multiply_monomials` in the future.

Reply:
I think it is better to use `#[allow(dead_code)]` and document why it is dead

Reply:
Done